core/src/main/scala/doobie/hi/preparedstatement.scala

```diff
@@ -72,7 +69,12 @@ object preparedstatement {
 
   /** @group Batching */
   val executeBatch: PreparedStatementIO[List[Int]] =
-    PS.executeBatch.map(_.toList)
+    PS.executeBatch.map(arr =>
+      // `Predef.intArrayOps` is not implicit as of Scala 2.12.0-M3
+      // https://github.com/scala/scala/blob/v2.12.0-M3/src/library/scala/Predef.scala#L336
+      // TODO replace with `_.toList` after it gets back
+      intArrayOps(arr).toList
```

Comment:
It's the only (temporary) fix needed for 2.12.0-M3, as well as in `c/s/m/s/d/hi/statement.scala`.